packages/scheduler/src/SchedulerPostTask.js

```diff
@@ -150,30 +151,28 @@ function runTask<T>(
           ),
           continuationOptions,
         )
-        .catch(handlePostTaskError);
+        .catch(handleAbortError);
     }
+  } catch (error) {
+    // We're inside a promise. If we don't handle this error, then it will
```

Comment:
```suggestion
    // We're inside a `postTask` promise. If we don't handle this error, then it will
```

Wasn't super clear which promise you we're talking about.